src/endpoints/v2/bugPost.ts

```diff
@@ -31,21 +35,21 @@ const handler: RequestHandler = async (req, res) => {
           {upsert:true}
         )
 
-        const result = await coll2.updateOne({userId:dUser.id, count: {$gt: 0}}, {$inc: {total: +1, count: -1}})
+        const result = await bugUsers.updateOne({userId:dUser.id, count: {$gt: 0}}, {$inc: {total: +1, count: -1}})
           
         if (result.modifiedCount === 0) return res.status(403).send('Too many active reports');
 
-        await coll.insertOne({
+        await bugs.insertOne({
           brief:req.body.brief,
           system:req.body.system,
           description:req.body.description,
           status: req.body.status,
           date: req.body.date,
           userName:dUser.username+"#"+dUser.discriminator,
           userId:dUser.id
-        });
-
-        res.sendStatus(200);
+        })
+        .then(result => res.sendStatus(200))
+        .catch(err => res.sendStatus(401));
```

Comment:
Don't think 401 applies to this here, I would send 500

Reply:
Updated